p4-16/psa/PSA.mdk

```diff
@@ -756,45 +778,43 @@ the contents of several metadata fields in the struct
             create a copy of the packet and send it to the packet
                 buffer with class of service specified by
                 ostd.clone_class_of_service, after which it will start
-                egress processing with egress_port=ostd.clone_port (which
-                might be PORT_CPU to send the clone to the control CPU).
-            The contents of the packet will be the same as the packet
-                as it comes out of the egress deparser, prepended with any
-                headers created by clone.emit() calls in the egress
-                deparser apply block.
+                egress processing.
```

Comment:
As earlier, the details removed are are I think better represented in the recently added 'initial values at beginning of egress processing' section.